packages/suite/src/components/guide/GuidePanel.tsx

```diff
@@ -12,19 +13,51 @@ import {
     Feedback,
 } from '@guide-views';
 import { useGuide } from '@guide-hooks';
+import * as guideActions from '@suite-actions/guideActions';
+import { GUIDE_ANIMATION_DURATION_MS } from '@suite/hooks/guide/useGuide';
```

Comment:
```suggestion
import { GUIDE_ANIMATION_DURATION_MS } from '@guide-hooks';
```

Reply:
Thanks you. Auto-import doesn't work properly for me then.

Reply:
Fixed auto impors by changing the alias order in the tsconfig